ui/src/components/data-table/data-table.tsx

```diff
@@ -53,11 +53,15 @@ export function DataTable<TData>({
 }: DataTableProps<TData>) {
   const pagination = table.getState().pagination;
   const totalPages = table.getPageCount();
+  // We need to extract the violating props from the props object
```

Comment:
ORT projects usually follow the convention to use passive voice in comments, commit messages etc. so you shoud better say "Extract violating properties ...". Also in pain text, "props" should be spelled out as "properties" unless it refers directly to the variable name.

Reply:
Ok, then let's stick with that...